index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <link rel="stylesseet" href="styles.css" >
+  </head>
+
+  <body>
+<section id ="welcome-section">
+  <h1> Texto </h1>
+</section>
+<section id ="projects">
+  <h2 class="project-tile"> Projects</h2>
+  <a href="www.google.com"> link</a>
+  </section>
+  <nav id="navbar">
+    <a href="#welcome-section"> link dentro de la pagina</a>
+  </nav>
```

Comment:
🙌🏼  I kindly recommend you create a section containing the nav tags within it.
The HTML section element (section) represents a generic section of a document. It is used to determine which content corresponds to which part of a schema. Think of the outline as the table of contents of a book; a common theme and related subsections. It is, therefore, a semantic tag.

```suggestion
<section id="nav"> 
  <nav id="navbar">
    <a href="#welcome-section"> link dentro de la pagina</a>
  </nav>
</section>

```